app/services/katello/puppet_class_importer_extensions.rb

```diff
@@ -1,14 +1,3 @@
-# rubocop:disable AccessModifierIndentation
```

Comment:
I know it says not to review, I just happened to see this and line 12 in this file and thought you might want to adjust your script to account for it.